README.md

```diff
@@ -24,6 +24,12 @@ Still here? To build the H3 C library, you'll need a C compiler (tested with `gc
 
 #### Install build-time dependencies
 
+* Alpine 
+```
+# Installing the bare build buid requirements
+apk add cmake make gcc libtool musl-dev g++
```

Comment:
now that @isaacbrodsky merged his PR, you can drop `g++`

Reply:
done